### DISCUSSION TITLE
Bookmark online backup

### DISCUSSION THREAD

**Discussion by stefaniecg** [organicmaps/organicmaps#5646] [Ideas]
I just lost all my **bookmarks** in organic maps because the app didn't start anymore, so i reinstalled it. 
Please allow a way to **back-up** the bookmarks because they are very valuable. 

Thank you very much !
Best regards,

Ps. 
I love organic Maps, thank you for your efforts to mantain it !
Keep up the good work.

**Comment by biodranik** [comment]
Please follow #622

Next time, if the app doesn't start, please get in touch with us first, and never uninstall it. There are some ways to help.
